tests/build-floating-ca-output.sh

```diff
@@ -0,0 +1,16 @@
+source common.sh
+
+sed -i 's/experimental-features .*/& ca-derivations/' "$NIX_CONF_DIR"/nix.conf
+
+nix build -f multiple-outputs.nix --arg floatingCA true --json a.all b.all | jq
```

Comment:
Is this first call intended?

Reply:
Oops good catch, that was just while debugging.